src/Http/Controllers/Support/ResolveController.php

```diff
@@ -78,48 +80,143 @@ public function resolveIdsToNames(Request $request)
 
         })->chunk(1000)->each(function ($chunk) use (&$response, $eseye) {
```

Comment:
Sounds a good idea. Can we put but excluded entries before chunking per 1000 ?
So we are certain to send optimized query to ESI.

Reply:
The actual `query` to ESI is only send in `resolveIDsfromESI`. I tried to stick to @leonjza 's resolver of fractions which was inside the chunk. 
I can try to alter this but there is not much need to imho

Reply:
sure, but with your change, it may send a query with less than 1000 IDs, since few of them may have been resolved with database stuff :)